config/database.yml

```diff
@@ -11,6 +11,10 @@ development:
   <<: *default
   database: simple-server_development
 
+profiling:
```

Comment:
Setting this to dev DB for now, but it may be nice to have this be separate so you could keep a large seed data set there.